book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -376,7 +376,6 @@ If the second part (the `path_to_where_to_put_file/file_name`) is substituted fo
 ## Volumes
 
 Every time a container is opened from an image, that container is completely new.
-Say a container is opened, and work is done within it.
 Say a container is opened, and work is done within it. If that container is closed, and the image it came from is again used to start another container, none of that work will be in the new one.
```

Comment:
```suggestion
Say a container is opened, and work is done within it. 
If that container is closed, and the image it came from is again used to start another container, none of that work will be in the new one.
```
Adding a semantic line break (as per [here](https://the-turing-way.netlify.app/community-handbook/style.html#write-each-sentence-in-a-new-line-line-breaks))